.gitignore

```diff
@@ -1,3 +1,7 @@
 build/
 lib/
 results/
+.DS_Store
+venv/
+__pycache__/
+.idea/

```

Comment:
nit: newline after `.idea/`

Reply:
.gitignore type corrected locally and pushed to https://github.com/bllewell/tpcc